x/recovery/spec/03_events.md

```diff
@@ -1,5 +1,8 @@
 <!--
-order: 4
+order: 3
+title: "Recovery events"
+parent:
+  title: "recovery"
 -->
 
 # Events
```

Comment:
Watch out for keeping the headline ordering consistent without skipping levels of hierarchy:

`### Recovery` -> `## Recovery`

Also all table cells should be left-aligned.